lib/stripe/account.ex

```diff
@@ -31,13 +31,99 @@ defmodule Stripe.Account do
 
   @valid_create_keys [:country, :email, :managed]
 
-  @valid_update_keys [
-    :business_logo, :business_name, :business_primary_color, :business_url,
-    :debit_negative_balances, :decline_charge_on, :default_currency, :email,
-    :external_account, :legal_entity, :metadata, :product_description,
-    :statement_descriptor, :support_email, :support_phone, :support_url,
-    :tos_acceptance, :transfer_schedule, :transfer_statement_descriptor
-  ]
+  @address_map %{
+    city: :string,
+    country: :string,
+    line1: :string,
+    line2: :string,
+    postal_code: :string,
+    state: :string
+  }
+
+  @address_kana_kanji_map %{ # Japan only
+    city: :string,
+    country: :string,
+    line1: :string,
+    line2: :string,
+    postal_code: :string,
+    state: :string,
+    town: :string
+  }
+
+  @dob_map %{
+    day: :integer,
+    month: :integer,
+    year: :integer
+  }
+
+  @update_schema %{
+    business_logo: :string,
+    business_name: :string,
+    business_primary_color: :string,
+    business_url: :string,
+    debit_negative_balances: :boolean,
+    decline_charge_on: %{
+      avs_failure: :boolean,
+      cvc_failure: :boolean
+    },
+    email: :string,
+    external_account: %{
+      object: :string, # required
+      account_number: :string, # required
+      country: :string, # required
+      currency: :string, # required
+      account_holder_name: :string,
+      account_holder_type: :string,
+      routing_number: :string
+    },
+    legal_entity: %{
+      address: @address_map,
```

Comment:
Missing `additional_owners` here, but I assume we also aren't yet sure how to implement those? We may want ti add an issue.